src/main/java/frc/robot/maps/Shrimp.java

```diff
@@ -0,0 +1,171 @@
+package frc.robot.maps;
+
+import org.littletonrobotics.junction.Logger;
+import org.littletonrobotics.junction.networktables.NT4Publisher;
+import org.littletonrobotics.junction.wpilog.WPILOGWriter;
+
+import com.chopshop166.chopshoplib.drive.SDSSwerveModule;
+import com.chopshop166.chopshoplib.drive.SDSSwerveModule.Configuration;
+import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.maps.SwerveDriveMap;
+import com.chopshop166.chopshoplib.motors.CSSpark;
+import com.chopshop166.chopshoplib.motors.CSSparkFlex;
+import com.chopshop166.chopshoplib.motors.CSSparkMax;
+import com.chopshop166.chopshoplib.sensors.CtreEncoder;
+import com.chopshop166.chopshoplib.sensors.gyro.PigeonGyro2;
+import com.chopshop166.chopshoplib.states.PIDValues;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.pathplanner.lib.util.HolonomicPathFollowerConfig;
+import com.pathplanner.lib.util.PIDConstants;
+import com.pathplanner.lib.util.ReplanningConfig;
+import com.revrobotics.CANSparkLowLevel.PeriodicFrame;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.PowerDistribution;
+import edu.wpi.first.wpilibj.PowerDistribution.ModuleType;
+
+@RobotMapFor("Shrimp")
+public class Shrimp extends RobotMap {
+
+    private static void setStatusPeriods(CSSpark motor, int status0, int status1, int status2) {
+        motor.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus0, status0);
+        motor.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus1, status1);
+        motor.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus2, status2);
+    }
+
+    @Override
+    public SwerveDriveMap getDriveMap() {
+
+        // Remember to divide by 360
+        // CAN ID 2
+        final double FLOFFSET = 0;
+        // CAN ID 4
+        final double FROFFSET = 0;
+        // CAN ID 1
+        final double RLOFFSET = 0;
+        // CAN ID 3
+        final double RROFFSET = 0;
+
+        // Value taken from CAD as offset from center of module base pulley to center
+        // of the robot
+        final double MODULE_OFFSET_XY = Units.inchesToMeters(6.00015);
+        final PigeonGyro2 pigeonGyro2 = new PigeonGyro2(1);
+
+        final CSSparkMax frontLeftSteer = new CSSparkMax(4);
```

Comment:
Can we re-order the CAN IDs so they're sequential. This was likely an artifact of rebuilding the modules or something.